backend/fms_core/template_importer/row_handlers/sample_qc/sample.py

```diff
@@ -29,14 +29,21 @@ def process_row_inner(self, sample, sample_information, process_measurement, pro
             if sample_obj.is_library:
                 self.errors['source_sample'] = f"Source sample can't be a library or a pool of libraries."
 
+            initial_volume = sample_information['initial_volume']
+            measured_volume = sample_information['measured_volume']
+
+            if initial_volume is None:
+                initial_volume = sample_obj.volume
```

Comment:
We need to validate that if initial_volume is none, measured_volume is none too. Otherwise we introduce false information the volume at the time of the submission might not be the same as at the time of the measurement.

Reply:
It's already done? unless `sample_obj.volume` can be None for some reason...